### DISCUSSION TITLE
Prune multiple apps like when using --filter?

### DISCUSSION THREAD

**Discussion by beeauvin** [vercel/turborepo#5655] [Help]
The docs don't suggest this is possible, and after trying a few things I am pretty sure it's not, but before I give up on the idea, is it? Specifically I want to be able to say something like, `turbo prune --scope=@backend` and have it grab all the backend apps the same way that `turbo build --filter="@backend/**"` would.

**Comment by anthonyshew** [comment]
The way that `turbo prune` is designed today likely wouldn't support this. `prune` dumps your prune to an `out` directory so multiple applications would step on each other.

Can I ask what your use case is for this?

**Comment by anthonyshew** [comment]
Will close due to staleness.
